functionary/core/models/environment_variable.py

```diff
@@ -0,0 +1,59 @@
+""" Package model """
+import uuid
+
+from django.db import models
+
+
+class EnvironmentVariable(models.Model):
```

Comment:
Somewhat late thought here, but how would you feel about calling this just `Variable`?  And then on the function model and package.yaml the field would be `variables` rather than `env_variables`?  The reason for this suggestion is that even knowing exactly what this is, it still takes my mind a second each time to disassociate the "Environment" in "EnvironmentVariable" from a Functionary "Environment."  I feel like removing "Environment" from the model name here leaves less room for confusion.

Reply:
Also worth mentioning is that if we go that route, the `variable_name` field here probably just becomes `name`.

Reply:
renamed